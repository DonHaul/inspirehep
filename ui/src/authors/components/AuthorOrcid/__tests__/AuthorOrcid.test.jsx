import React from 'react';
import { render } from '@testing-library/react';

import AuthorOrcid from '../AuthorOrcid';

describe('AuthorOrcid', () => {
  it('renders with orcid', () => {
    const orcid = '0000-0001-8058-0014';
    const { asFragment } = render(<AuthorOrcid orcid={orcid} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
