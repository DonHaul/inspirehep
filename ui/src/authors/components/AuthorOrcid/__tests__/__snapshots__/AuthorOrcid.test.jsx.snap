// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorOrcid renders with orcid 1`] = `
<DocumentFragment>
  <a
    class="__AuthorOrcid__"
    href="//orcid.org/0000-0001-8058-0014"
    target="_blank"
  >
    <img
      alt="ORCID"
      src="orcid.svg"
    />
  </a>
</DocumentFragment>
`;
